internal/file/opener.go

```diff
@@ -7,7 +7,8 @@ import (
 
 // Opener is an object that stores a path to later be opened as a file.
 type Opener struct {
-	path string
+	path   string
+	digest string
```

Comment:
this seems convenient but out of place for what an Opener is meant to do

Reply:
👍  Moved into the archive parser